docs/en/latest/config.json

```diff
@@ -26,11 +26,18 @@
       "items": [
         "tutorials/expose-api",
         "tutorials/protect-api",
-        "tutorials/observe-your-api",
+        {
```

Comment:
I don't know how it displays but I like the idea of grouping tutorials by themes

Reply:
Yes, it is good to group them when we have 3 related topics.